schema/hotel-booking.json

```diff
@@ -485,5 +486,10 @@
         }
     ],
     "attributes": {
+        "canonicalGroupingPeriod": {
+            "type": "number",
+            "description": "Canonical grouping period in minute",
+            "value": 1440
```

Comment:
`1440` seems rather large, a user may book multiple hotels when planning a trip over the course of a day.